blog2epub/crawlers/wordpress.py

```diff
@@ -145,6 +144,7 @@ def replace_images(self):
                 self.html = self.html.replace(
                     "<!--#blog2epubimage#" + image + "#-->", image_html
                 )
+        self.content = self.html
```

Comment:
Also this line I wasn't 100% on. It feels a little hacky but I wasn't sure of a better way to make this change. Maybe if the image processing happens entirely over `article.content`, but I wasn't sure if there was a meaningful difference between the two that might be seriously affected.

Reply:
Thanks for this contribution, I appreciate it very much!
I spent a lot of time preparing the Android version, I completely neglected crawlers, which have many "hacky" places.

I will merge this commit to the v1.4.0 branch and test this change. Looks reasonable.